app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class QuotesController < ApplicationController
+    def index
+        @data = QuoteService.quote_search(params[:query])
+        @quotes = QuoteFacade.get_quotes(params[:query])
```

Comment:
The `get_quotes` method in your facade class is calling the same `quote_search` method in your service, so you are hitting the same API endpoint twice. I will add some thoughts about a more efficient way to handle this down in the comments for the facade class.